base/show.jl

```diff
@@ -1832,7 +1832,7 @@ function show_sym(io::IO, sym::Symbol; allow_macroname=false)
         print(io, '@')
         show_sym(io, Symbol(sym_str[2:end]))
     else
-        print(io, "var", repr(string(sym))) # TODO: this is not quite right, since repr uses String escaping rules, and Symbol uses raw string rules
+        print(io, "var\"$(escape_raw_string(string(sym)))\"")
```

Comment:
```suggestion
        print(io, "var\"", escape_raw_string(string(sym)), '"')
```
It doesn't really make any actual difference, but avoiding interpolation is slightly more idiomatic in Base.